config/locales/en.yml

```diff
@@ -1509,6 +1509,8 @@ en:
     transaction:
       commission_payment_name: "Commission payment for %{listing_title}"
       commission_payment_description: "Marketplace %{service_name} took this commission from transaction regarding %{listing_title}"
+    wait_while_loading: "Please wait."
+    chatting_with_paypal: "We need to chat with Paypal."
```

Comment:
This generates quite an uncertain feeling - should it be just: We are chatting with PayPal.

Otherwise, I like it though ;)